src/twisted/trial/newsfragments/9962.misc

```diff
@@ -0,0 +1 @@
+remove _twisted.trial.util:runSequentially
```

Comment:
```suggestion
remove twisted.trial.util._runSequentially
```

Reply:
However, content of `.misc` news fragments is ignored, so an empty file might be better.

Reply:
> is there a spec of when the trac ticket/news fragment is needed?

Yes, it's linked from the checklist template.